roles/os10_route_map/tests/main.yaml

```diff
@@ -53,3 +61,9 @@ os10_route_map:
            add: aa
            delete: aa
       state: present
+    - name: as-path-test
+      permit: true
+      seq_num: 10
+      match:
+        - as_path: bb
+      state: present
```

Comment:
To be perfectly honest I don't understand why `match` is supposed to have a list of elements - since you can't really have multiple statements in a route map use the same `seq_num` - however the fact that `match` is a sequence of elements was decided upon by the authors of this role, not myself, so I am merely adding `as_path` as an additional supported match type. 

A possible re-design would be to do something like:

```yaml
route_map:
  - name: my route map
    sequences:
     - seq: 10
       match:
           as_path: bbb
     - seq: 20
       match:
           ip_prefix: aaaa
```

Or something to that effect, but that would be a radical change that would break everything and is probably not possible to change.